atomic_reactor/plugin.py

```diff
@@ -354,6 +396,44 @@ def __init__(self, dt, workflow, plugins_conf, *args, **kwargs):
         self.plugins_results = workflow.prebuild_results
         super(PreBuildPluginsRunner, self).__init__(dt, workflow, 'PreBuildPlugin', plugins_conf, *args, **kwargs)
 
+
+class BuildStepPlugin(BuildPlugin):
+    pass
+
+
+class BuildStepPluginsRunner(BuildPluginsRunner):
+
+    def __init__(self, dt, workflow, plugin_conf, *args, **kwargs):
+        logger.info("initializing runner of build-step plugin")
+        self.plugins_results = workflow.buildstep_result
+
+        if not plugin_conf:
+            # if none buildstep plugins specified, fallback to docker api plugin
+            plugin_conf = [{'name': 'docker_api', 'is_allowed_to_fail': False}]
+        else:
```

Comment:
Don't think there should be an `else` here. If "falsy", just define `plugin_conf` as `[{'name': 'docker_api'}]` and let it run through normal execution. 

Right now, it won't set `required` to False as it would be if `plugin_conf` was "truthy".